bin/wmagent-component-standalone

```diff
@@ -15,18 +15,25 @@ wmagent-component-standalone -c $WMA_CONFIG_FILE -p JobAccountant -e $WMA_ENV_FI
 
 import os
 import sys
+sys.dont_write_bytecode = True
```

Comment:
I'm not sure I see reason for this introduction, can you elaborate more why you add it?

Reply:
Todor, thanks for explanation, I strongly suggest that you add this (shorter) description as docstring before this line to explain this reason. We will forget about it in a future but it would be extremely useful to anyone who may comeback to this code to understand this reason.

Reply:
done